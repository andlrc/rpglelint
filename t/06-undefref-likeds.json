[
   {
      "column":21,
      "file":"t/06-undefref-likeds.rpgle",
      "line":2,
      "message":"'undef_ds' undeclared",
      "severity":"warning"
   }
]
