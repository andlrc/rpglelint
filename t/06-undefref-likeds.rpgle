// rpglelint: -Wundefref
dcl-ds my_ds likeds(undef_ds);
