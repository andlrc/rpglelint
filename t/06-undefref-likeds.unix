t/06-undefref-likeds.rpgle:2:21: warning: 'undef_ds' undeclared [-Wundefined-reference]
 dcl-ds my_ds likeds(undef_ds);
                     ^
