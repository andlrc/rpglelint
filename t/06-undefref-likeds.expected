t/06-undefref-likeds.rpgle:2:21: warning: 'undef_ds' undeclared [-Wundefref]
 dcl-ds my_ds likeds(undef_ds);
                     ^
